pinot-common/src/main/java/org/apache/pinot/common/function/FunctionUtils.java

```diff
@@ -31,6 +31,9 @@
   private FunctionUtils() {
   }
 
+  // TODO: Do we need to create a class for handling JSON. It doesn't seem like this is needed since JSON type directly
```

Comment:
Currently we use `String` to represent JSON. If we change that to use `JsonNode`, then we can add `JsonNode` here. One problem of adding `JsonNode` is that it is an abstract class, and won't match the actual class

Reply:
Removed comment. I think String is good enough for now.